packages/express/src/index.test.ts

```diff
@@ -0,0 +1,103 @@
+import LogtoClient from '.';
+import { testMiddleware } from './test-utils';
+import { LogtoExpressConfig } from './types';
+
+const signInUrl = 'http://mock-logto-server.com/sign-in';
+
+const configs: LogtoExpressConfig = {
+  appId: 'app_id_value',
+  endpoint: 'https://logto.dev',
+  baseUrl: 'http://localhost:3000',
```

Comment:
Don't we need to support `cookieSecret` in express sdk?

Reply:
In express, middlewares usually only do one thing, so we leave it to other middleware named "express-session", will mention this in docs.